config/dependency/kuberay-operator/kustomization.yaml

```diff
@@ -4,9 +4,6 @@ kind: Kustomization
 # The workaround is to use kustomize namespace override to create under aibrix-system namespace.
 
 resources:
-- crds/ray.io_rayclusters.yaml
```

Comment:
Is this redundant?

Reply:
We have two make commands, `make install and make deploy`. Both commands use kubectl create because of apply does not work if the CRD manifest is more than 261244 bytes. Currently, ray CRDs are installed as part of `make deploy`, with this change ray CRDs will be installed as part of `make install using kubectl create` and we can use `kubectl apply in make deploy`. 

Users need to run make install once and using kubectl create is OK. But make deploy need to be run with every version change or config change, so using kubectl apply is necessary for make deploy.